ups/meas_extensions_simpleShape.table

```diff
@@ -5,10 +5,11 @@
 # included by low-level LSST packages such as utils or daf_base.
 # Any other package whose interface is used should be listed explicitly
 # rather than assuming it will be included recursively.
-setupRequired(meas_algorithms)
+setupRequired(meas_base)
```

Comment:
Just to be clear. I also see dependencies on numpy, utils, pex_config and pex_exceptions.